app/models/modules/priceable.rb

```diff
@@ -0,0 +1,5 @@
+module Priceable
+  def convert_to_dollars
+    self.unit_price = (self.unit_price.to_f / 100)
+  end
+end
```

Comment:
@mel-rob - hello

Reply:
My initial thought is that I would change the call-back that you are using to `before_create`. It doesn't look like you are actually using any validations, so maybe that method is never being invoked.

Reply:
Awesome I will try that, thanks!